containers/init/build/csharp/build_qps_worker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+bash /src/workspace/tools/run_tests/helper_scripts/pre_build_csharp.sh
```

Comment:
this script is considered to be part of run_tests.py's internals/implementation detail so I wouldn't rely on its exact semantics here (btw, you're also setting MSBUILD_CONFIG and CONFIG env variables in the Dockerfile to match the semantics, which feels hacky)

I think it's better to just "inline" the interesting contents of "pre_build_csharp.sh" since the only interesting command is the cmake command: https://github.com/grpc/grpc/blob/e745f8cb558798f1c8b91baa42d46228a9d4a0c4/tools/run_tests/helper_scripts/pre_build_csharp.sh#L24

Basically, just do this (it's more consistent, readable and also simpler)
```
# Build C# native extension with cmake
mkdir -p cmake/build
pushd cmake/build
cmake -DgRPC_BUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Release ../..
make grpc_csharp_ext
popd

cd src/csharp
dotnet publish Grpc.IntegrationTesting.QpsWorker/ -c Release -f netcoreapp2.1 -o ../../../qps_worker
```

Reply:
Agreed, this make the Dockerfile  much clearer, thank you for reviewing. I will make the change accordingly.